raiden/tests/unit/api/test_python_api.py

```diff
@@ -156,3 +160,55 @@ def test_api_channel_events(raiden_chain):
         channel_0_1.channel_address, max_block + 1, max_block + 100
     )
     assert not results
+
+
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+@pytest.mark.parametrize('number_of_nodes', [2])
+@pytest.mark.parametrize('number_of_tokens', [1])
+@pytest.mark.parametrize('register_tokens', [True, False])
+@pytest.mark.parametrize('settle_timeout', [600])
+def test_channel_open_close(raiden_network, token_addresses, settle_timeout):
+    app0, app1 = raiden_network
+
+    maker_address = app0.raiden.address
+    taker_address = app1.raiden.address
```

Comment:
`taker` is even more confusing. You don't "take" or "decline" a channel. If somebody opens a channel with you, there will be a channel. Maybe use `participant0` and `participant1` instead.